libs/explorers/comparison-tools/src/lib/legend-panel/legend/legend.component.html

```diff
@@ -0,0 +1,19 @@
+<h2>COLOR</h2>
```

Comment:
Unlike in Agora, which is mostly just an image, this component will allow us to customize the content for any explorer app's Legend dialog.